website/docs/03_architecture/06_read-model.mdx

```diff
@@ -310,6 +310,11 @@ export class MessageReadModel {
 }
 ```
 
+## Projecting fields from a Read Model
```

Comment:
In the docs, I would change the word `Project` and its derivatives to "Selecting fields" or similar. The reason is that we already have the concept "Project" associated to read model projections, and it can be confusing to the users if we also use the same word for this. 

Besides, in the code we use `.select`, so it makes sense to talk about is using the "Select fields" expresion

Reply:
@alvaroloes changed in commit c4b7ca6c. Also, in the same commit, a warning message was added indicating the limitations of selecting fields inside arrays, related to [this comment](https://github.com/boostercloud/booster/pull/1481#discussion_r1616552349)

Reply:
Great!